From jluebbe@lasnet.de Fri Oct 27 07:59:57 2006
From: Jan Luebbe <jluebbe@lasnet.de>
Subject: USB: sierra: Fix id for Sierra Wireless MC8755 in new table
To: linux-usb-devel@lists.sourceforge.net
Cc: Kevin Lloyd <linux@sierrawireless.com>, Greg Kroah-Hartman <gregkh@suse.de>
Content-Type: text/plain
Date: Fri, 27 Oct 2006 18:59:24 +0200
Message-Id: <1161968364.4638.4.camel@polaris>
Mime-Version: 1.0

From: Jan Luebbe <jluebbe@lasnet.de>

The new version of sierra.c has introduced tables for the 1 port and 3
port variants. The device id i added in my last patch needs to be added
to the 3 port table.

Signed-off-by: Jan Luebbe <jluebbe@lasnet.de>
Cc: Kevin Lloyd <linux@sierrawireless.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/usb/serial/sierra.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/usb/serial/sierra.c
+++ gregkh-2.6/drivers/usb/serial/sierra.c
@@ -62,6 +62,7 @@ static struct usb_device_id id_table_3po
 	{ USB_DEVICE(0x1199, 0x0218) },	/* Sierra Wireless MC5720 */
 	{ USB_DEVICE(0x1199, 0x6802) },	/* Sierra Wireless MC8755 */
 	{ USB_DEVICE(0x1199, 0x6803) },	/* Sierra Wireless MC8765 */
+	{ USB_DEVICE(0x1199, 0x6804) },	/* Sierra Wireless MC8755 for Europe */
 	{ USB_DEVICE(0x1199, 0x6812) },	/* Sierra Wireless MC8775 */
 	{ USB_DEVICE(0x1199, 0x6820) },	/* Sierra Wireless AirCard 875 */
 	{ }
